code/game/objects/items/toys.dm

```diff
@@ -1495,3 +1495,43 @@
 	if(cooldown <= world.time)
 		cooldown = (world.time + 10)
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, src, 'sound/effects/blobattack.ogg', 50, FALSE), 0.5 SECONDS)
+
+
+/*
+ * Eldrich Toys
+ */
+
+/obj/item/toy/eldrich_book
+	name = "Codex Cicatrix"
+	desc = "A toy book that closely resembles the Codex Cicatrix. Covered in fake polyester human flesh and has a huge goggly eye attached to the cover. The runes are gibberish and cannot be used to summon demons... Hopefully?"
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "book"
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb_continuous = list("sacrifices", "transmutes", "graspes", "curses")
+	attack_verb_simple = list("sacrifice", "transmute", "grasp", "curse")
+	var/open = FALSE
```

Comment:
Give your new variable a short documentation on what it does, what it is about or for.
This one should be rather simple.